modules/ROOT/pages/deployment/wopi/wopi.adoc

```diff
@@ -0,0 +1,103 @@
+= Office Applications using WOPI
+:toc: right
+:wopi_subdir: /ocis_wopi
+:download-gh-directory-url: https://download-directory.github.io
+:traefik_url: https://traefik.io/traefik/
+:nginx_url: https://www.nginx.com
+:apache_url: https://www.apache.org
+
+:description: Infinite Scale uses the WOPI protocol to integrate office applications. This document describes the general flow and basic settings used behind.
+
+== Introduction
+
+{description} For more details on possible configuration values see the xref:conf-examples/office-integration.adoc[Office Integration] document at the configuration examples. 
+
+For the time being, the supported office applications are:
+
+* Collabora
+* OnlyOffice
+
+== General Information
+
+Infinite Scale can be configured to use office applications via a browser in an iframe. Depending on the use case and among other things, administrators can:
+
+* provide multiple applications from different vendors in parallel,
+* define which icons are used for which applications,
+* define which app is preferred to open a file.
+
+Configured applications can be selected by users when opening a document via the WebUI. The list of available applications to open a file show an icon and its app name.
+
+Office applications are :
+
+* provided via the xref:{s-path}/app-provider.adoc[App-Provider Service] because they are not able to register themself, see the xref:#wopi-coniguration-examples[docker-copmpose.yml] file. 
+
+* configured for the use with the WebUI via the xref:{s-path}/app-registry.adoc[App-Registry Service]. Here, the graphical representation details like icons, app names, mimetypes, app to open etc is set, see the xref:#wopi-coniguration-examples[app-registry.yaml] file.
```

Comment:
This is not correct. The mimetypes are registered in the app provider by calling the discovery endpoint of the office application. The app registry is only needed to set the default app.

Reply:
> The app registry is only needed to set the default app

This is not correct. `app-registry` defines which apps are available for which mimetypes, which icons they show, which names they get, which app is default when not overrided by the user etc and all that info is shown as list items in the webUI for the user whe he selects to open a file. See the example below.

```
  - mime_type: application/vnd.oasis.opendocument.text
    extension: odt
    name: OpenDocument
    description: OpenDocument text document
    icon: ''
    default_app: Collabora
    allow_creation: true
```
This was part of the intense discussion with Willy.

Reply:
Still not correct. The app registry config is optional. It works without that. There are two different things:

1) Registering the mimetype -> done via autodiscovery https://onlyoffice.ocis-wopi.latest.owncloud.works/hosting/discovery
2) Providing the actions for the new button and the default app settings. -> app-registry config

Reply:
@mmattel Can you clarify this on a follow up PR?

I think it might be confusing to see more mimetypes attached to apps than provided by the app-registry.yaml

The app-registry.yaml only configures a subset for the "+ New" action. The File open Action and the logo for the file open action are defined otherwise.

Reply:
sure

Reply:
Note that sources like app-registry is/was provided by @wkloucek